src/line-chart/line-chart.component.ts

```diff
@@ -175,7 +178,6 @@ export class LineChartComponent extends BaseChartComponent {
   @Input() yAxisTickFormatting: any;
   @Input() roundDomains: boolean = false;
   @Input() tooltipDisabled: boolean = false;
-  @Input() showSeriesOnHover: boolean = true;
```

Comment:
Why did we take it out? Why not set it to false and let users decide what to do about it?